tutor/src/components/radio-input.jsx

```diff
@@ -55,7 +55,7 @@ const StyledRadioInput = styled.input.attrs( () => ({
 `;
 
 const RadioInput = observer((props) => {
-  const [field] = useField({ type: 'text', ...props });
+  const [field] = props.standalone ? [] : useField({ type: 'text', ...props });
```

Comment:
Let's it be used outside of a formik context, but happy to hear suggestions on a different approach.

Reply:
yeah I think that's fine.